shell/platform/windows/flutter_windows_view.cc

```diff
@@ -127,6 +129,14 @@ void FlutterWindowsView::SendWindowMetrics(size_t width,
   auto result = FlutterEngineSendWindowMetricsEvent(engine_->engine(), &event);
 }
 
+void FlutterWindowsView::SendInitialBounds() {
+  // Initial bounds must be sent after engine has initialized
```

Comment:
This comment appears to be describing things that a client of this class is responsible for, so should be in the header somewhere, not here.

Reply:
fixed in next commit